public_suffix_list.dat

```diff
@@ -12900,6 +12900,10 @@ nh-serv.co.uk
 // Submitted by Jeff Wheelhouse <support@nearlyfreespeech.net>
 nfshost.com
 
+// NFT.Storage : https://nft.storage/
+// Submitted by Vasco Santos <vasco.santos@protocol.ai>
```

Comment:
If possible, please make this a non-personal address so it stays reachable.

Reply:
added support email as an alternative like other entries in the list :)